source/client-side-encryption/tests/README.rst

```diff
@@ -961,12 +964,13 @@ Test cases
 
 .. note::
 
-   IMPORTANT: If csfle_ is visible to the operating system's library search
-   mechanism, the expected server error generated by these
+   IMPORTANT: If crypt_shared_ is visible to the operating system's library
```

Comment:
And earlier comment said:

> The MongoDB Enterprise distribution includes a dynamic library named ``mongo_crypt_v1``...

If that's the actual name of the shared library, should we use that when referring to a dynamic library (at least in this section)?

Reply:
Resolving per https://github.com/mongodb/specifications/pull/1236#discussion_r886020343